sections/kubecontext.zsh

```diff
@@ -17,6 +17,7 @@ SPACESHIP_KUBECONTEXT_SUFFIX="${SPACESHIP_KUBECONTEXT_SUFFIX="$SPACESHIP_PROMPT_
 SPACESHIP_KUBECONTEXT_SYMBOL="${SPACESHIP_KUBECONTEXT_SYMBOL="☸️  "}"
 SPACESHIP_KUBECONTEXT_COLOR="${SPACESHIP_KUBECONTEXT_COLOR="cyan"}"
 SPACESHIP_KUBECONTEXT_NAMESPACE_SHOW="${SPACESHIP_KUBECONTEXT_NAMESPACE_SHOW=true}"
+SPACESHIP_KUBECONTEXT_COLORGROUPS=(${SPACESHIP_KUBECONTEXT_COLORGROUPS=()})
```

Comment:
just to confirm, is it typo that you have `=(${...})` instead of `="${...}"`?

Reply:
No, it's intentional: https://github.com/denysdovhan/spaceship-prompt/pull/576#discussion_r238070644

Reply:
just saw your answer in a different thread, so should this be just =${...}?

Reply:
Nope, it has to be surrounded by parentheses on assignment, otherwise it's just a string value:
```zsh
$ var1=(aa bbb ccc)
$ var2=${var1=()}
$ echo $var2
aa bbb ccc
$ echo $#var2
10
```

Reply:
```zsh
$ echo ${(t)var2}
scalar
```

Reply:
Gotcha, okay thanks for confirming 🙂

Reply:
I mean, I tried them all. With and without quotes, parentheses, curly braces, mixed of them... I took a while to figure the syntax 😁